tests/x11regressions/gnomecase/login_test.pm

```diff
@@ -23,6 +23,7 @@ sub auto_login_alter {
     send_key "super";
     wait_still_screen;
     type_string "settings", 1;    #Use '1' to give gnome-shell enough time to search settings module; Otherwise slow worker will cause failed result.
```

Comment:
the comment is btw nonsense. The concept of '1' was most likely copied from a send_key call - where it's actually abused often. But in type_string the 1 means type like a grandma types with her nose. So just remove the 1 here - then the type_string will finish before gnome-shell had a chance to blink :)

For extra safety you can add a sleep 1 after the type_string.